### PR TITLE
Windows additions to jenkinsfile

### LINKED ISSUE #68: Windows additions to Jenkinsfile
Make the necessary editions to the Jenkinsfile for builds to run on the Windows build node.

**Outputs:**
- 3 working Windows Jenkins pipelines
    - `Windows-10-2018b`
    - `PR-Windows-10-2018b`
    - `Branch-Windows-10-2018b`

### PR DESCRIPTION
**Pull request by hsautessella**
This PR adds PowerShell commands to the Jenkinsfile to build, test and package the Windows version of Herbert.

Alongside this PR, the `PR-Windows-10-2019b`, `Branch-Windows-10-2019b` and the nightly `Windows-10-2019b` Jenkins pipelines have been created.

A test was failing locally and on Jenkins so https://github.com/pace-neutrons/Herbert/pull/69/commits/aba253308d0f36e21845dfd5d18dd3de6c2a2742 disables it. This should be fixed and re-enabled - the test doesn't appear to fail on Linux.

Fixes #68

### CODE REVIEW

File: _test/CMakeLists.txt [hunk 1]
```diff
@@ -36,8 +36,12 @@ foreach(_test_dir ${TEST_DIRECTORIES})
     # The MPI tests require that `herbert_on.m` be on the path for all workers
     # when Matlab starts up: add the path of the directory it's copied into
     # to the MATLABPATH environment variable when tests are run.
+    set(MATLABPATH "${MATLABPATH}:${CMAKE_BINARY_DIR}/herbert_config")
```
Comment (nickbattam-tessella) [inline _test/CMakeLists.txt]: this is done in every loop; the path variable can be constructed outside the loop and reused

File: _test/CMakeLists.txt [hunk 2]
```diff
@@ -36,8 +36,12 @@ foreach(_test_dir ${TEST_DIRECTORIES})
...
+    if(WIN32)
+        string(REPLACE ":" ";" MATLABPATH "${MATLAB_PATH}")
```
Comment (nickbattam-tessella) [inline _test/CMakeLists.txt]: Widows paths containing "C:\" will break this substitution

File: admin/CMakeLists.txt [hunk 3]
```diff
@@ -6,9 +6,11 @@ if(${BUILD_TESTS})
 endif()
 configure_file("worker_v2.m.template" "${WORKER_CONFIG_DIR}/worker_v2.m")
 
+set(_path_regex "[a-zA-Z0-9/\\\\_\\-\\.: ]+")
```
Comment (nickbattam-tessella) [inline admin/CMakeLists.txt]: can you add a comment explaining what this regex matches?

Comment (hsautessella) [reply admin/CMakeLists.txt]: Fixed in #71

**Pull request merged**